services-custom/s3-transfer-manager/README.md

```diff
@@ -0,0 +1,49 @@
+## Overview
+
+This project provides a much improved experience for S3 customers needing to easily perform uploads and downloads of
+objects to and from S3 by providing the S3 S3TransferManager, a high level
+library built on the [AWS Common Runtime S3 Client](https://github.com/awslabs/aws-crt-java).
+
+## Getting Started
+
+### Initialization
+You can instantiate the transfer manager easily using the default settings
```

Comment:
Do we need to mention  xml snippet of the maven dependencies  ?

Reply:
Yeah, good call. Will add